lib/whats/actions/send_hsm_message.rb

```diff
@@ -3,11 +3,11 @@
 module Whats
   module Actions
     class SendHsmMessage
-      PATH = "/api/rest_send.php"
+      PATH = "/v1/messages"
 
-      def initialize(client, username, namespace, element_name, params)
+      def initialize(client, wa_id, namespace, element_name, params)
         @client       = client
-        @username     = username
+        @wa_id        = wa_id
```

Comment:
`wa_id = whatsapp_id` foi so para ficar mais curto?

Reply:
`wa_id` is the name of the key used in the http response from WhatsApp, just keeping the same name in our implementation